### PR TITLE
#120: API-core-parallel_dispatch-ParallelReduceTag from md to rst

### PR DESCRIPTION
**Pull request by antoinemeyer5**
### THIS PR:
* API/core/parallel-dispatch/ParallelReduceTag transition from .md to .rst

before (.md) | after (.rst) | after 2 (.rst) [curent code]
:----------------: | :-------------: | :-------------:
before_1 | after_1 | image

### CODE REVIEW

**Review by fnrizzi — APPROVED** [review]

File: docs/source/API/core/parallel-dispatch/ParallelReduceTag.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_ExecPolicy.hpp>``
+
+.. _text: ../parallel-dispatch/parallel_reduce.html
+
+.. |text| replace:: ``parallel_reduce``
+
+A tag used in team size calculation functions to indicate that the functor for which a team size is being requested is being used in a |text|_
```
Comment (nmm0) [inline docs/source/API/core/parallel-dispatch/ParallelReduceTag.rst:13]: Might be easier/more readable to use a cross reference to parallel_reduce here

**Review by nmm0 — APPROVED** [review]
Looks good to me!

**Comment by fnrizzi** [comment]
@antoinemeyer5 please address the link comment

File: docs/source/API/core/parallel-dispatch/ParallelReduceTag.rst (hunk 1 above)
Comment (antoinemeyer5) [reply docs/source/API/core/parallel-dispatch/ParallelReduceTag.rst]: done

**Pull request merged**